packages/core/src/components/date-picker/date-picker.scss

```
@@ -270,6 +271,7 @@
       }
 
       &.week-day {
+        @include ellipsis;
```

```suggestion
        @include ellipsis;
        display: inline-block;
        align-content: center;
```
'text-overflow: ellipsis' only works for 'block' elements.